porter/v1/addons.proto

```diff
@@ -151,6 +151,8 @@ message Newrelic {
   optional bool prometheus_enabled = 8;
   // pixie_enabled determines whether pixie is enabled
   optional bool pixie_enabled = 9;
+  // kube_events_enabled determines whether tracking for kube events is enabled
+  optional bool kube_events_enabled = 10;
```

Comment:
curious if there'd be any way to give these fields names that more closely correspond to what the user would gain from enabling them if they're not familiar.

maybe not possible / maybe only at the porter.yaml level?

Reply:
Yeah I feel like that's only worth doing at the porter.yaml level, my view of proto is it's just the language between porter and ccp, and anyone trying to update an addon will hit the translation layer in porter